velox/docs/functions/spark/decimal.rst

```diff
@@ -9,6 +9,9 @@ https://cwiki.apache.org/confluence/download/attachments/27362075/Hive_Decimal_P
 
 https://msdn.microsoft.com/en-us/library/ms190476.aspx
 
+Additionally, the computation of decimal division adapts to the allowing precision loss parameter,
```

Comment:
I still feel this sentence is confusing. My understanding is that allow-precision-loss flag applies to all operations, not just division. This sentence suggests that allow-precision-loss flag applies to division only. Am I missing something?

CC: @rui-mo

Reply:
Right. The allow-precision-loss flag applies to all operations, while there are two processes: 1) calculate the result precision and scale based on certain formula 2) adjust them if the calculated precision exceeds 38. Here I assume it refers to the first step, where only division is affected by the flag. All operations are affected by this flag in the second step as L62-L75 describes. Perhaps we could describe the two-step process clearer.